net/ServerSocket.hpp

```diff
@@ -103,14 +103,11 @@ class ServerSocket : public Socket
         if (events & POLLIN)
         {
             std::shared_ptr<Socket> clientSocket = accept();
-            if (!clientSocket)
+            if (clientSocket)
             {
-                const std::string msg = "Failed to accept. (errno: ";
-                throw std::runtime_error(msg + std::strerror(errno) + ')');
-            }
-
-            LOG_TRC("Accepted client #" << clientSocket->getFD());
-            _clientPoller.insertNewSocket(std::move(clientSocket));
+                LOG_TRC("Accepted client #" << clientSocket->getFD() << ", " << *clientSocket);
+                _clientPoller.insertNewSocket(std::move(clientSocket));
+            } // else continue this ServerSocket as accept() already determined whether to throw (ending it).
```

Comment:
Comment reads oddly - if there is a clear warning in the accept() method for this case lets ignore adding this here.

Reply:
OK